Fixed UnicodeDecodeError: 'utf-8' codec can't decode byte 0xe2 in position 3444: invalid continuation byte #2

zaghadon opened the pull request:
Changed EML File Read Mode Encoding to Latin-1

zaghadon committed:
Fixed UnicodeDecodeError: 'utf-8' codec can't decode byte 0xe2 in position 3444: invalid continuation byte #2

Changed EML File Read Mode Encoding to Latin-1

zaghadon commented:
This is a Quick Fix for Issue #2 and It's up for comments and review.

dunnkers commented:
Hi @zaghadon ! This is awesome. Can you add an `.eml` file to the test data in the `test_emails` folder that shows a failing unit test?

zaghadon committed:
Added Test Email with non UTF-8 Characters which breaks execution.

In response to https://github.com/dunnkers/eml-to-html/pull/3#issuecomment-2066455497 on #3

zaghadon committed:
Replaced email address to trigger github action

zaghadon commented:
Hi @dunnkers I added the test email, the workflow doesn't break on my end because of the fix, I guess I need to push it to a different branch entirely and open a pull request for it first.
